sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-coe-requests-report.sql

```diff
@@ -0,0 +1,62 @@
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'COE_Requests',
+        'SELECT
+            student_user.first_name AS "Student First Name",
+            student_user.last_name AS "Student Last Name",
+            sin_validation.sin AS "SIN",
+            application.data ->> ''studentNumber'' AS "Student Number",
+            student_user.email AS "Student Email Address",
+            student.contact_info ->> ''phone'' AS "Student Phone Number",
+            application.application_number AS "Application Number",
+            TO_CHAR((assessment.assessment_date AT TIME ZONE ''America/Vancouver''), ''YYYY-MM-DD HH24:MI:SS'') AS "Assessment Date",
+            program.program_name AS "Program Name",
+            offering.offering_name AS "Offering Name",
+            offering.offering_intensity AS "Study Intensity",
+            student.disability_status AS "Profile Disability Status",
+            assessment.workflow_data -> ''calculatedData'' ->> ''pdppdStatus'' AS "Application Disability Status",
+            TO_CHAR(offering.study_start_date, ''YYYY-MM-DD'') AS "Study Start Date",
+            TO_CHAR(offering.study_end_date, ''YYYY-MM-DD'') AS "Study End Date",
+            disbursement.coe_status AS "COE Status",
+            TO_CHAR((disbursement.coe_updated_at AT TIME ZONE ''America/Vancouver''), ''YYYY-MM-DD HH24:MI:SS'') AS "COE Actioned",
+            disbursement.tuition_remittance_requested_amount AS "Remittance Requested",
+            disbursement.tuition_remittance_effective_amount AS "Remittance Disbursed",
+            (
+                SELECT
+                    SUM(disbursement_value.value_amount)
+                FROM
+                    sims.disbursement_values disbursement_value
+                WHERE
+                    disbursement_value.disbursement_schedule_id = disbursement.id
+                    AND disbursement_value.value_code != ''BCSG''
+            ) AS "Estimated Disbursement Amount",
+            CASE
+                WHEN disbursement.disbursement_schedule_status IN (''Sent'', ''Ready to send'') THEN TO_CHAR(disbursement.date_sent, ''YYYY-MM-DD'')
+                ELSE TO_CHAR(disbursement.disbursement_date, ''YYYY-MM-DD'')
+            END AS "Disbursement Date"
```

Comment:
Would the below be a simplification or the idea was to enforce the choice by the status?
```sql
TO_CHAR(COALESCE(disbursement.disbursement_date, disbursement.date_sent), ''YYYY-MM-DD'') AS "Disbursement Date"
```

Reply:
When both exist, `date_sent` must be preferred, hence I will use 
```sql
COALESCE(disbursement.date_sent, disbursement.disbursement_date)
```